README.md

```diff
@@ -162,10 +164,10 @@ tenant init.
 ## CQL
 
 For CQL support *all* fields recognized must be explicitly defined. Undefined CQL
-fields are rejected. Example to get titles:
+fields are rejected. Example to get books:
 
 ```
```

Comment:
@adamdickmeiss I would drop this code section and instead reference the file in the example directly, like so https://www.stevemar.net/github-code-in-readme/

Reply:
Pointing to the project of its own with dangling links is not my cup of tee. I'm happy to refer to relative links to THIS project which is going to be valid always and is not depending on GitHub btw.